src/OpenTelemetry.Instrumentation.Http/CHANGELOG.md

```diff
@@ -2,7 +2,10 @@
 
 ## Unreleased
 
-* Dropped `netstandard2.0` target and added `net6.0`
+* Dropped `netstandard2.0` target and added `net6.0`. .NET 5 reached EOL
+  in May 2022 and .NET Core 3.1 reaches EOL in December 2022. The
```

Comment:
https://dotnet.microsoft.com/download/dotnet nit: add this link right here so people can quickly check the dates themselves.

Reply:
Good idea. Added links to prior changelog announcements for ASP.NET Core and the API package as well.